docs/cloud-native-application-developers-manual.md

```diff
@@ -0,0 +1,60 @@
+# Cloud Native Application Developers Manual
+
+This document is intended for cloud native application authors targeting application deployments on Bitnami Kubernetes Production Runtime (BKPR). It describes the configuration an application author needs to perform for automating the Ingress, TLS, logging and monitoring support for their application.
```

Comment:
an application author ----> support for his or her application (singular consistency)